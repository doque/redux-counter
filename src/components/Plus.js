import React from "react";
import Button from "./styled/Button";

const Plus = ({ increment }) =>
  <Button up onClick={increment}>
    +
  </Button>;

export default Plus;
