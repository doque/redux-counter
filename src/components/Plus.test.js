import React from "react";
import renderer from "react-test-renderer";
import "jest-styled-components";

import Plus from "./Plus";

describe("Plus", () => {
  it("renders correctly and looks amazing", () => {
    const tree = renderer.create(<Plus large />).toJSON();
    expect(tree).toMatchStyledComponentsSnapshot();
  });
});
