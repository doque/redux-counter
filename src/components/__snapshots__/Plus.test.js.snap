// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Plus renders correctly and looks amazing 1`] = `
.kbbzHL {
  font-size: 20px;
  background: mediumseagreen;
  border: 1px solid red;
  border-radius: 2px;
  -webkit-flex-grow: 1;
  -ms-flex-grow: 1;
  flex-grow: 1;
}

<button
  className="sc-bdVaJa kbbzHL"
  onClick={undefined}
>
  +
</button>
`;
